databuilder/snippets/dsl.py

```diff
@@ -0,0 +1,24 @@
+# :snippet imports
```

Comment:
thought: I guess that it would be possible to separate out examples into subdirectories if need be (because for instance, we might want a self-contained project with `project.yaml` or there are conflicting behaviours between different modules?) 

I've not tested this though.

Reply:
Not sure I follow the example but breaking them out into subdirectories could be tricky, snex has some hard code behaviour on name mapping, however it's an open source tool so we could potentially fork if it was necessary.